src/control/system/member.go

```diff
@@ -83,14 +83,14 @@ type Members []*Member
 // MemberResult refers to the result of an action on a Member identified
 // its string representation "address/rank".
 type MemberResult struct {
-	ID     string
+	Rank   uint32
```

Comment:
I wonder if it makes sense to make this an ioserver.Rank instead, to avoid all of the casting. I don't feel strongly about it, just something that occurred to me.

Reply:
you still have to then cast in and out of Protobuf messages